gcloud/bigquery/table.py

```diff
@@ -525,3 +551,146 @@ def delete(self, client=None):
         """
         client = self._require_client(client)
         client.connection.api_request(method='DELETE', path=self.path)
+
+    def data(self, max_results=None, page_token=None, client=None):
```

Comment:
Given the use of a token and maxResults (paging semantics), I'm not sure `data` is the best name.